tests/integration/cartography/intel/aws/ec2/test_internet_gateway.py

```diff
@@ -0,0 +1,32 @@
+import cartography.intel.aws.ec2
+import tests.data.aws.ec2.internet_gateway
+
+TEST_ACCOUNT_ID = '012345678912'
+TEST_REGION = 'us-east-1'
+TEST_UPDATE_TAG = 123456789
+
+
+def test_load_internet_gateways(neo4j_session):
+    data = tests.data.aws.ec2.internet_gateway.DESCRIBE_GATEWAYS
+    cartography.intel.aws.ec2.internet_gateways.load_internet_gateways(
+        neo4j_session,
+        data,
+        TEST_REGION,
+        TEST_ACCOUNT_ID,
+        TEST_UPDATE_TAG,
+    )
+
+    expected_nodes = {
+        "igw-1234XXX",
+        "igw-7e3a7c18",
+        "igw-f1c81494",
+    }
+
+    nodes = neo4j_session.run(
+        """
+        MATCH (n:AWSInternetGateway) RETURN n.id;
```

Comment:
Can you add another test for 

```
(:AWSInternetGateway)<-[:RESOURCE]-(:AWSAccount)
```
?

Reply:
I noticed that there seemed to be a few instances where we needed to set up an account in as part of our tests, so I created a common function for that. Let me know if that isn't the direction you wanted to head.

Reply:
Sure, that works!